pkgs/applications/networking/browsers/firefox/wrapper.nix

```diff
@@ -320,18 +337,13 @@ let
         # preparing for autoconfig
         mkdir -p "$out/lib/${firefoxLibName}/defaults/pref"
 
-        cat > "$out/lib/${firefoxLibName}/defaults/pref/autoconfig.js" <<EOF
-          pref("general.config.filename", "mozilla.cfg");
-          pref("general.config.obscure_value", 0);
-        EOF
+        echo 'pref("general.config.filename", "mozilla.cfg");' >> "$out/lib/${firefoxLibName}/defaults/pref/autoconfig.js"
```

Comment:
Is there a reason for touching this part?
The only semantic change I see here, is appending instead of (potentially) overwriting `autoconfig.js`.. and given its nature (its content should be very much fixed for our purposes) I find overwriting it clearer.